Dockerfile

```diff
@@ -1,7 +1,11 @@
 FROM jboss/wildfly:8.2.0.Final
 MAINTAINER Democracy Works, Inc. <dev@democracy.works>
 
+ENV HOST_IP "TO.BE.FILLED.IN"
+
 COPY ./start /bin/start
 
+ADD standalone-rsyslog.xml /opt/jboss/wildfly/standalone/configuration/standalone-rsyslog.xml
+
 ENTRYPOINT ["/bin/start"]
-CMD ["-b", "0.0.0.0", "-bmanagement", "0.0.0.0"]
+CMD ["-c", "standalone-rsyslog.xml", "-Djboss.host.name=$HOST_IP", "-b", "0.0.0.0", "-bmanagement", "0.0.0.0"]
```

Comment:
The default docker hostnames look like random ID's. Try setting the hostname with the `--hostname` flag in the docker run and see if that shows up in the logs w/o doing anything else.